ert_shared/storage/extraction_api.py

```diff
@@ -264,8 +258,13 @@ def dump_to_new_storage(reference=None, rdb_connection=None, blob_connection=Non
             )
         )
 
-    rdb_connection.close()
-    blob_connection.close()
+    except:
+        rdb_session.rollback()
```

Comment:
After finally

Reply:
what..? we want only having rollback if the dump fails right? this is not a test... 😬

Reply:
Me stupid. Yes